.vscode/settings.json

```diff
@@ -5,15 +5,15 @@
     "public/": false
   },
   "search.exclude": {
+    "**/__generated__": false,
```

Comment:
I thought about excluding stuff in this folder from fuzzy match / search, but realized that that would then limit the ability to do TypeScript auto-imports, e.g.,: 

<img width="665" alt="Screen Shot 2019-04-21 at 10 56 52 PM" src="https://user-images.githubusercontent.com/236943/56485437-d0a03180-6488-11e9-8264-a96483343ff6.png">

Reply:
(Also note the `SpaceProps` in there -- from `@types/styled-system`. Every styled-system function has a interface like this that you can import. I would suggest going into yr UI folder and typing all of your primitives following a pattern similar to Palette's [`<Flex>` component](https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Flex/Flex.tsx#L46-L90)